cmd/fetch/fetch.go

```diff
@@ -282,9 +283,39 @@ func Command() *cobra.Command {
 	cmdutil.RegisterNameFilterFlags(cmd)
 	cmdutil.RegisterMetricsFlags(cmd)
 
+	cmd.PersistentFlags().Var(
+		enumflag.NewWithoutDefault(&tableHandlingMode, "string", TableHandlingOptionStringRepresentations, enumflag.EnumCaseInsensitive),
+		"table-handling",
+		fmt.Sprintf("the way to handle the table initialization on the target database: %q(default), %q or %q",
+			noneTableHandlingKey,
+			dropOnTargetAndRecreateTableHandlingKey,
+			truncateIfExistsTableHandlingKey,
+		),
+	)
+
 	if err := cmd.PersistentFlags().MarkHidden(testOnlyFlagStr); err != nil {
 		panic(err)
 	}
 
 	return cmd
 }
+
+type TableHandlingOption enumflag.Flag
+
+const (
+	None TableHandlingOption = iota
+	DropOnTargetAndRecreate
+	TruncateIfExists
+)
+
+const (
+	noneTableHandlingKey                    = "none"
+	dropOnTargetAndRecreateTableHandlingKey = "drop-on-target-and-recreate"
+	truncateIfExistsTableHandlingKey        = "truncate-if-exist"
```

Comment:
```suggestion
	truncateIfExistsTableHandlingKey        = "truncate-if-exists"
```
Just to match the const.